File: C/URI/1009.c
#include <stdio.h>
 
int main() {
 
 char nome[255];
 double venda, salario;

        scanf("%s %lf %lf", nome, &salario, &venda);
        venda = venda * 0.15;
        salario = venda + salario;
        printf("TOTAL = R$ %.2lf\n", salario);

 
    return 0;
}

File: JS/URI/1009.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


var nome = lines.shift();
var salario = parseFloat(lines.shift());
var vendas = parseFloat(lines.shift());

var total = salario+(vendas*0.15);

console.log("TOTAL = R$", total.toFixed(2));